frontends/election-manager/src/screens/reports_screen.tsx

```diff
@@ -173,9 +173,12 @@ export function ReportsScreen(): JSX.Element {
   const ballotPrintingSummaryText = (
     <p>
       <strong>
-        {pluralize('official ballot', totalBallotsPrinted, true)}{' '}
+        {pluralize(
+          `${format.count(totalBallotsPrinted)} official ballots`,
```

Comment:
Why move the count into the string rather than use the `pluralize` function's built-in way to do so, as we were doing before?

Reply:
Localization?

Reply:
Mostly because using `format.count` will add grouping separators as appropriate.